README.md

```diff
@@ -1,3 +1,71 @@
 # Hey Rodney dataset
 
 This is a dataset of voice samples for our wake word detection of "Hey Rodney".
+
+## Steps to Record audio data and upload it on github
+
+## Specifications:
+
+| __Phrase__ | "Hey Rhodney"   |
+| ------------- | ------------- |
+| __Number of Samples__  | 3  |
+| __Sample Rate__ | 16 KHz  |
+| __Number of Channels__ | 1 |
+| __Encoding__ | Signed 16-bit PCM |
+
+
+
+## Step 1: Record the Audio
+
+### Steps to implement in audacity:
+1. Open audacity 
```

Comment:
Where do I get "Audacity" from?

Reply:
I will add the steps to download audacity